core/src/main/java/nu/yona/server/subscriptions/entities/UserPrivate.java

```diff
@@ -180,4 +180,9 @@ public void touch()
 	{
 		touchVersion++;
 	}
+
+	public UUID getUserAnonymizedID()
```

Comment:
Shouldn't this be the replacement of ``getVPNLoginID``? In other words: shouldn't ``getVPNLoginID`` be removed?